### DISCUSSION TITLE
Prism library enforces inconsistent dependencies ?

### DISCUSSION THREAD

**Discussion by sigmarsson** [unoplatform/uno#11745] [Q&A]
Adding the library to a WinUI desktop application requires the package ```Uno.Extensions.Hosting.WinUI``` to be added. Is it regular ?

<img width="592" alt="image" src="https://user-images.githubusercontent.com/43340959/226303541-2593005b-ed83-487c-a874-90e97f50ecf2.png">

**Comment by nickrandolph** [comment]
@sigmarsson which template are you using?

**Reply by sigmarsson** [reply]
Created again with Visual Studio 17.4 template, new Uno App and selected Windows and Android only.  Now there is a different error : 


<img width="629" alt="image" src="https://user-images.githubusercontent.com/43340959/226407430-bbc5c0d9-542b-4fc7-b12a-453fa3fa3691.png">

<img width="305" alt="image" src="https://user-images.githubusercontent.com/43340959/226407686-246f9f73-39d1-4e59-8a67-342f272e94a1.png">

(Prism.net7.0-windows10.0.18362)

How could I vanish this error please ?

```
C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(2352,5): warning MSB3277: Found conflicts between different versions of "Microsoft.Windows.SDK.NET" that could not be resolved.
1>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(2352,5): warning MSB3277: There was a conflict between "Microsoft.Windows.SDK.NET, Version=10.0.22000.24, Culture=neutral, PublicKeyToken=31bf3856ad364e35" and "Microsoft.Windows.SDK.NET, Version=10.0.22621.24, Culture=neutral, PublicKeyToken=31bf3856ad364e35".
1>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(2352,5): warning MSB3277:     "Microsoft.Windows.SDK.NET, Version=10.0.22000.24, Culture=neutral, PublicKeyToken=31bf3856ad364e35" was chosen because it was primary and "Microsoft.Windows.SDK.NET, Version=10.0.22621.24, Culture=neutral, PublicKeyToken=31bf3856ad364e35" was not.
1>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(2352,5): warning MSB3277:     References which depend on "Microsoft.Windows.SDK.NET, Version=10.0.22000.24, Culture=neutral, PublicKeyToken=31bf3856ad364e35" [C:\Users\weath\.nuget\packages\microsoft.windows.sdk.net.ref\10.0.22000.27\lib\net6.0\Microsoft.Windows.SDK.NET.dll].
```

**Reply by jeromelaban** [reply]
See this article: https://platform.uno/docs/articles/features/winapp-sdk-specifics.html#adjusting-windows-sdk-references. You need to align the Framework References.

**Reply by sigmarsson** [reply]
Thank you. It has really helped.

**Reply by sigmarsson** [reply]
@nickrandolph btw I had to add the package ```Uno.Extensions.Hosting.WinUI``` 2.3.1 to launch the application. I do not know if it is correct or not.

**Reply by nickrandolph** [reply]
@dansiegel is there any guidance/docs for using the Prism Uno template?

**Reply by sigmarsson** [reply]
@nickrandolph do you reckon the other issue https://github.com/unoplatform/uno/discussions/11741 is also related to the inappropriate template utilized ?
